frontend/src/slices/trucks/selectors.ts

```diff
@@ -0,0 +1,23 @@
+import { type DataStatus } from '~/libs/enums/enums.js';
+import {
+  type RootState,
+  type TruckEntityT,
+  type ValueOf,
+} from '~/libs/types/types.js';
+import { type GetAllTrucksByUserIdResponseDto } from '~/packages/trucks/trucks.js';
+
+const selectDataStatus = (state: RootState): ValueOf<typeof DataStatus> => {
+  return state.trucks.dataStatus;
+};
+
+const selectTrucks = (state: RootState): GetAllTrucksByUserIdResponseDto =>
+  state.trucks.trucks;
+
+const selectTruck =
+  (truckId: number | null) =>
+  (state: RootState): TruckEntityT | undefined =>
```

Comment:
can it be TruckEntityT | null? I'd prefer to use null instead of undefined because they have different concepts.
Null - stands for No data, while undefined is about 'something is not defined yet'